UE/Application/SmsDb.cpp

```diff
@@ -9,4 +10,16 @@ void SmsDb::addReceivedSms(const Sms& sms)
     smsMessages.emplace_back(sms, SmsState::NotViewed);
 }
 
+const SmsDb::SmsMessages &SmsDb::getSmsMessages() {
```

Comment:
Style rest of code is not using auto func() -> T syntax. I am not sure it is good idea to use it only in this one place, without changing syntax in rest of function in repo

Reply:
see 2nd example in [ES.11: Use auto to avoid redundant repetition of type names](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#es11-use-auto-to-avoid-redundant-repetition-of-type-names)

Reply:
and look [here](https://stackoverflow.com/questions/52103775/advantage-of-using-trailing-return-type-in-c11-functions). There are cases where you HAVE to use this syntax. You can't really ban. Additionally this style has to be used when you want to specify lambda's return type.